tests/cursorid/cursorid-set_state_error-001.phpt

```diff
@@ -0,0 +1,18 @@
+--TEST--
+MongoDB\Driver\CursorId::__set_state() requires "id" string field
+--FILE--
+<?php
+
+require_once __DIR__ . '/../utils/basic.inc';
+
+echo throws(function() {
+    MongoDB\Driver\CursorId::__set_state(['id' => 0]);
+}, 'MongoDB\Driver\Exception\InvalidArgumentException'), "\n";
```

Comment:
This can be `MongoDB\Driver\Exception\InvalidArgumentException::class` instead. Many of the older test files didn't use that since the syntax was not supported in all versions of PHP that the driver itself supported. This is no longer an issue.

See: https://www.php.net/manual/en/language.oop5.basic.php#language.oop5.basic.class.class for more context

Another benefit to this approach is that a typo in the class name leads to a more helpful error that the class cannot be found, rather than `throws()` simply failing some internal string comparison.